results/source_data/fig1_case_maps.csv

```diff
@@ -1,55 +1,55 @@
 "","adm_name","lon","lat","cases","country","date"
-"3826","Washington",-120.441,47.382,8383,"USA",2020-04-06
```

Comment:
@hdruckenmiller is there maybe some randomization in the ordering of this fig1 csv? I don't think any of the actual information changed, but the tests were raising an error b/c it looks like things got shuffled around. one number in the other figure source_data changed too which might be from some small usafacts update, but I think this one just looks shuffled